pkgs/applications/science/misc/simgrid/default.nix

```diff
@@ -18,13 +18,12 @@ in
 
 stdenv.mkDerivation rec {
   name = "simgrid-${version}";
-  version = "3.21";
+  version = "3.22.2";
 
-  src = fetchFromGitHub {
-    owner = "simgrid";
-    repo = "simgrid";
-    rev = "v${replaceChars ["."] ["_"] version}";
-    sha256 = "1v0dwlww2wl56ms8lvg5zwffzbmz3sjzpkqc73f714mrc9g02bxs";
+  src = fetchgit {
```

Comment:
There's `fetchFromGitLab`. You just need to supply `domain = "framagit.org"`.
That'll download the tarball directly without needing to fetch all the git stuff.

Reply:
Thanks, I did not know about `fetchFromGitlab`. Should be fixed in aa29b79e8a2.